daal4py/sklearn/neighbors/knn.py

```diff
@@ -30,6 +30,22 @@
 import logging
 
 
+def configure_algorithm(classifier, fptype, params):
+    if classifier.algorithm == 'brute':
+        train_alg = d4p.bf_knn_classification_training
+        predict_alg = d4p.bf_knn_classification_prediction
+        # Brute force method always computes in doubles due to precision need
+        compute_fptype = 'double'
```

Comment:
Please explain.
cc @Alexsandruss

Reply:
sklearn use same strategy - computation for brute-force always uses float64